test/simulation/install.sh

```diff
@@ -1,46 +1,17 @@
 #!/bin/bash
+set -ex
+source .env
 
-delete_dir() {
-	if ! rm -rf  $1 >/dev/null 2>&1; then
-		echo "unable to delete directory $1"
-		exit 1
-	fi
-	return 0
-}
+# cleanup existing data directories to prevent errors
+# during consecutive runs
+rm -Rf $TEMP_PATH/xuddatadir*
```

Comment:
Would it make sense to do this in `clean.sh` or something similar? It just seems a little out of place to have cleanup code at the beginning of an install script.

I know that these data dirs get cleaned by the `TearDown` function when `XudCleanup` is true, but I also know that when the tests don't exit gracefully sometimes they don't get deleted. But it's also somewhat nice to be able to deliberately keep the data folders, in case you want to inspect it afterwards.